src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheck.java

```diff
@@ -29,12 +29,14 @@
  * Checks that the clone method is not overridden from the
  * Object class.
  * </p>
- *
- * <p>Rationale: The clone method relies on strange/hard to follow rules that
- * do not work it all situations.  Consequently, it is difficult to
- * override correctly.  Below are some of the rules/reasons why the clone
- * method should be avoided.
```

Comment:
Why remove the last sentence and just place it before we start the list?

Reply:
done

Reply:
@pbludov Please point to me where `Below are some of the rules/reasons why the clone method should be avoided.` was re-added?